symfony/framework-bundle/3.3/Makefile

```diff
@@ -1,31 +1,31 @@
-CONSOLE := $(shell printf %q "`which bin/console`")
+CONSOLE := $(shell which bin/console)
```

Comment:
That's what we had before, but that does not work well if there is a space in your directory names.

Reply:
ah, sorry, I see that you've changed the code using $CONSOLE as well. nvm

Reply:
Why not `test -e` when using it bool'sch only?